src/Compilers/Test/Resources/Core/CompilerTestResources.csproj

```diff
@@ -346,6 +346,7 @@
     <EmbeddedResource Include="SymbolsTests\snKey2.snk" />
     <EmbeddedResource Include="SymbolsTests\snPublicKey.snk" />
     <EmbeddedResource Include="SymbolsTests\snPublicKey2.snk" />
+    <EmbeddedResource Include="SymbolsTests\snMaxSizeKey.snk" />
```

Comment:
Generating a key of this size takes a *really* long time -- like 20 min on my box